test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -192,13 +189,22 @@ def reply_to_message_at_index(self, index: int, message: str):
         message_object_to_reply_to = self.get_message_at_index(index)
         verify(not is_null(message_object_to_reply_to), "Message to reply to is loaded")
         move_mouse_over_object(message_object_to_reply_to)
-        click_obj_by_name(ChatComponents.REPLY_TO_MESSAGE_BUTTON.value)
+        foundreplyToButton = getChildrenWithObjectName(message_object_to_reply_to, "replyToMessageButton")[0]
```

Comment:
Could we add `replyToMessageButton` as a constant like we do with `ChatComponents`? Maybe we can add a new enum called `ChildObjectNames` ?? Or a better naming.. I cannot think of another now! :)

Reply:
Done